README.md

```diff
@@ -185,6 +185,27 @@ const myRoute = {
 
 > This will use `MyCustomValue` as the view name.
 
+## Using with composition API
+
+In order to use this plugin with composition api (inside your `setup` method), you can just call the custom composable `useGtm`.
+
+Example:
+
+```js
+import {useGtm} from 'vue-gtm';
+
+const gtm = useGtm();
+
+gtm.trackEvent({
```

Comment:
Because this is (I think) especially for a `setup` method, should we create an example including this `setup` method so it's more real-world example?

Reply:
Yes sure. I updated the example to be more detailed.